### PR TITLE
update readme document

### PR DESCRIPTION
**Pull request by BellaXiang**
Signed-off-by: BellaXiang <42797797+BellaXiang@users.noreply.github.com>

### What problem does this PR solve?
Update the Readme document

### What is changed and how does it work?
- update the blog list
- adjust the structure
- add the "media coverage" part
- fix URL of "chaos-mesh-action: Integrate Chaos Engineering into Your CI" blog

### CODE REVIEW

**Comment by BellaXiang** [comment]
@dcalvin @cwen0 PTAL~

**Review by cwen0 — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -67,15 +67,16 @@ This is a list of known adopters of Chaos Mesh. Some have already gone into prod
...
 ## Blogs
 
 - [Chaos Mesh - Your Chaos Engineering Solution for System Resiliency on Kubernetes](https://pingcap.com/blog/chaos-mesh-your-chaos-engineering-solution-for-system-resiliency-on-kubernetes/)
 - [Run Your First Chaos Experiment in 10 Minutes](https://pingcap.com/blog/run-first-chaos-experiment-in-ten-minutes/)
 - [Simulating Clock Skew in K8s Without Affecting Other Containers on the Node](https://pingcap.com/blog/simulating-clock-skew-in-k8s-without-affecting-other-containers-on-node/)
+- [Chaos Mesh® Joins CNCF as a Sandbox Project](https://chaos-mesh.org/blog/chaos-mesh-join-cncf-sandbox-project)
+- [Building an Automated Testing Framework Based on Chaos Mesh® and Argo](https://chaos-mesh.org/blog/building_automated_testing_framework)
+- [chaos-mesh-action: Integrate Chaos Engineering into Your CI](https://chaos-mesh.org/blog/chaos-mesh-action-integrate-chaos-engineering-into-your-ci)
+- [Chaos Mesh 1.0: Chaos Engineering on Kubernetes Made Easier](https://chaos-mesh.org/blog/chaos-mesh-1.0-chaos-engineering-on-kubernetes-made-easier)
+- [Chaos Mesh X Hacktoberfest 2020 - An Invitation to Open Source](https://chaos-mesh.org/blog/chaos-mesh-x-hacktoberfest-2020)
```
Comment (dcalvin) [inline README.md:79]: Blogs 
```suggestion
Blogs on Chaos Mesh design & implementation, features, chaos engineering, community updates, etc.  See [Chaos Mesh Blogs](https://chaos-mesh.org/blog). Here are some recommended ones for you to start with:

- [Chaos Mesh - Your Chaos Engineering Solution for System Resiliency on Kubernetes](https://pingcap.com/blog/chaos-mesh-your-chaos-engineering-solution-for-system-resiliency-on-kubernetes/)
- [Run Your First Chaos Experiment in 10 Minutes](https://pingcap.com/blog/run-first-chaos-experiment-in-ten-minutes/)	- [Run Your First Chaos Experiment in 10 Minutes](https://pingcap.com/blog/run-first-chaos-experiment-in-ten-minutes/)
- [Simulating Clock Skew in K8s Without Affecting Other Containers on the Node](https://pingcap.com/blog/simulating-clock-skew-in-k8s-without-affecting-other-containers-on-node/)	- [Simulating Clock Skew in K8s Without Affecting Other Containers on the Node](https://pingcap.com/blog/simulating-clock-skew-in-k8s-without-affecting-other-containers-on-node/)
- [Building an Automated Testing Framework Based on Chaos Mesh® and Argo](https://chaos-mesh.org/blog/building_automated_testing_framework)
- [Chaos Mesh 1.0: Chaos Engineering on Kubernetes Made Easier](https://chaos-mesh.org/blog/chaos-mesh-1.0-chaos-engineering-on-kubernetes-made-easier)
```

Comment (BellaXiang) [reply README.md]: Done

**Review by dcalvin — APPROVED** [review]
LGTM

**Review by fewdan — APPROVED** [review]
LGTM

**Pull request merged**